deploy/cloudformation/elastic-agent-ec2.yml

```diff
@@ -4,7 +4,7 @@ Description: Creates an EC2 instance with permissions to run elastic-agent
 Parameters:
   LatestAmiId:
     Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
-    Default: '/aws/service/canonical/ubuntu/server/jammy/stable/current/arm64/hvm/ebs-gp2/ami-id'
+    Default: '/aws/service/ami-amazon-linux-latest/amzn2-ami-minimal-hvm-arm64-ebs'
```

Comment:
> As Amazon Linux AMIs are patched and new versions are released to the public, AWS updates the Parameter Store value with the latest ImageID value for each AMI namespace in all supported Regions.

[source](https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/#:~:text=As%20Amazon%20Linux%20AMIs%20are%20patched%20and%20new%20versions%20are%20released%20to%20the%20public%2C%20AWS%20updates%20the%20Parameter%20Store%20value%20with%20the%20latest%20ImageID%20value%20for%20each%20AMI%20namespace%20in%20all%20supported%20Regions.)

i think we could use a fixed version by date instead of always latest, but not sure we should as latest tag is probably the safest, which is the aim of this PR